Sources/IR/Emitter.swift

```diff
@@ -750,8 +761,8 @@ public struct Emitter {
     }
   }
 
-  /// Emits `expr` as an operand suitable to be the callee of a `CallInstruction`, updating `conventions`
-  /// and `arguments` if `expr` refers to a bound member function.
+  /// Emits the IR of `expr` as an operand suitable to be the callee of a `CallInstruction` into
+  /// `module`, updating `conventions` and `arguments` if `expr` refers to a bound member function.
```

Comment:
```suggestion
  /// Inserts the IR of lambda expression `expr` into `module`, updating `conventions` and `arguments` if `expr` refers to a bound member function.
```

But you can't just say "updating."  What specific changes are made?